.github/pull_request_template.md

```diff
@@ -0,0 +1,65 @@
+Fixes #XXXX.
+
+(Replace XXXX with the id of the issue fixed in this PR. Remove the above line if there is no corresponding
+issue. Don't reference the issue in the title of this pull-request.)
+
+(If you are a committer, follow the PR action item checklist for committers:
+https://github.com/apache/incubator-druid/blob/master/dev/committer-instructions.md#pr-and-issue-action-item-checklist-for-committers.)
+
+### Description
+
+Describe the goal of this PR, what problem are you fixing. If there is a corresponding issue (referenced above), it's
+not necessary to repeat the description here, however, you may choose to keep one summary sentence.
+
+Describe your patch: what did you change in code? How did you fix the problem?
+
+If there are several relatively logically separate changes in this PR, create a mini-section for each of them. For
+example:
+#### Fixed the bug ...
+#### Renamed the class ...
+#### Added a forbidden-apis entry ...
+
+In each section, please describe design decisions made, including:
+ - Choice of algorithms
+ - Behavioral aspects. What configuration values are acceptable? How are corner cases and error conditions handled, such
+   as when insufficient resources are available?
```

Comment:
I would rephrase "as when there are insufficient resources" or "when not enough resources are available".

Reply:
Changed to "as when there are insufficient resources".